starter code/app.js

```diff
@@ -108,42 +108,42 @@ const newObject = array => ({
 
 
 
-let sum = function(a, b, c, d) {
+let sum = ((a, b, c, d) => {
   return a + b + c + d;
-};
+});
```

Comment:
Really nice work here! On step 10, it mentions that you can rewrite these on one line where possible. This challenge is a good example of when you can write it inline:

`let sum = (a, b, c, d) => a + b + c +d ;`